Add troubleshooting to README

JeffReeves opened the pull request:
This addresses Issues #3 and #7:

- https://github.com/artanikin/vim-synthwave84/issues/3

- https://github.com/artanikin/vim-synthwave84/issues/7

JeffReeves committed:
Add troubleshooting to README

This addresses Issues #3 and #7:

- https://github.com/artanikin/vim-synthwave84/issues/3

- https://github.com/artanikin/vim-synthwave84/issues/7

JeffReeves committed:
Update README to use relative paths on images

Ruby, JavaScript, and Vim script now use relative paths on images

artanikin commented:
Thank you for your PR.
I intentionally use an absolute path for the images.
The sites [https://vimawesome.com/plugin/vim-synthwave84](https://vimawesome.com/plugin/vim-synthwave84) and [https://vimcolorschemes.com/artanikin/vim-synthwave84](https://vimcolorschemes.com/artanikin/vim-synthwave84) do not display images with relative paths. 
Could you change the relative path to the absolute path in your PR?

JeffReeves committed:
Change image paths to absolute paths

Absolute paths necessary for listing plugin on sites such as:
- https://vimawesome.com/plugin/vim-synthwave84
- https://vimcolorschemes.com/artanikin/vim-synthwave84

JeffReeves commented:
Thanks for review and the details on why the absolute paths are necessary. 
I've created a new commit with the image paths changed to absolute paths on your GitHub URL.

Please let me know if any other edits may be necessary.
